### ISSUE TITLE
Improve the Classic editor and Cloudflare notification modals

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#9428]
Both the modals use styles inherited from core `edit.css`. Not necessarily bad, as they're both in the `compat.php` file which is specific to WordPress.

However, the inherited styles make them not centered vertically. [Edit:] not perfectly horizontally centered too, because of box-sizing issues.

Also, the Cloudflare modal doesn't get focus. Initial focus should be set within the modal.

Lastly, when they become "full screen" on small screens, the content is cut off on the right.

![screen shot 2018-08-29 at 13 47 24](https://user-images.githubusercontent.com/1682452/44786816-9f8b8d80-ab95-11e8-8520-ca0af979971d.jpg)

![screen shot 2018-08-29 at 13 49 38](https://user-images.githubusercontent.com/1682452/44786817-9f8b8d80-ab95-11e8-80fb-3532a178de63.jpg)

![screen shot 2018-08-29 at 14 14 20](https://user-images.githubusercontent.com/1682452/44786943-014bf780-ab96-11e8-86b9-d52b134482b0.png)
